src/components/account/accountComponent.test.js

```diff
@@ -1,72 +1,106 @@
+        },
+        data: {
+          currentPeer: 'localhost',
+          port: 4000,
+          options: {
+            name: 'Custom Node',
+          },
+        },
+      },
+      account: {
+        isDelegate: false,
+        address: '16313739661670634666L',
+        username: 'lisk-nano',
+        balance: 1e8,
       },
-    },
-  };
-  const testAccount = {
-    isDelegate: false,
-    address: '16313739661670634666L',
-    username: 'lisk-nano',
-    balance: 1e8,
-  };
+    };
+  });
 
   it(' should render 3 article tags', () => {
-    const wrapper = shallow(<AccountComponent account={testAccount} peers={peers}
-      onActivePeerUpdated={onActivePeerUpdated} />);
+    const wrapper = shallow(<AccountComponent {...props} />);
     expect(wrapper.find('article')).to.have.lengthOf(3);
   });
 
   it('depicts being online when peers.status.online is true', () => {
-    const onlinePeers = Object.assign({}, peers, { status: { online: true } });
-    const wrapper = shallow(<AccountComponent account={testAccount} peers={onlinePeers}
-      onActivePeerUpdated={onActivePeerUpdated} />);
+    props.peers.status.online = true;
+    const wrapper = shallow(<AccountComponent {...props} />);
     const expectedValue = 'check';
     expect(wrapper.find('.material-icons').text()).to.be.equal(expectedValue);
   });
 
   it('should render balance with ClickToSend component', () => {
     const wrapper = mount(<Provider store={store}>
-      <AccountComponent account={testAccount} peers={peers}
-        onActivePeerUpdated={onActivePeerUpdated} />
+      <AccountComponent {...props} />
     </Provider>);
     expect(wrapper.find('.balance').find(ClickToSend)).to.have.lengthOf(1);
   });
 
   describe('componentDidMount', () => {
+    let accountApiMock;
+
+    beforeEach(() => {
+       accountApiMock = mock(accountApi);
+    });
+
+    afterEach(() => {
+      accountApiMock.restore();
+    });
+
     it('should be called once', () => {
       const actionSpy = spy(AccountComponent.prototype, 'componentDidMount');
-      mount(<Provider store={store}><AccountComponent account={testAccount} peers={peers}
-        onActivePeerUpdated={onActivePeerUpdated} /></Provider>);
+      mount(<Provider store={store}><AccountComponent {...props} /></Provider>);
       expect(actionSpy).to.have.been.calledWith();
     });
 
     it('binds listener to beat event', () => {
       const actionSpy = spy(document, 'addEventListener');
-      mount(<Provider store={store}><AccountComponent account={testAccount} peers={peers}
-        onActivePeerUpdated={onActivePeerUpdated} /></Provider>);
+      mount(<Provider store={store}><AccountComponent {...props} /></Provider>);
       expect(actionSpy).to.have.been.calledWith();
     });
+
+    it('calls props.onActivePeerUpdated', () => {
+      accountApiMock.expects('getAccountStatus').resolves({ success: true });
+      const wrapper = mount(<Provider store={store}><AccountComponent {...props} /></Provider>);
```

Comment:
You defined wrapper but you never used it. If it is extra, you need to remove it.